style.css

```diff
@@ -17,18 +17,18 @@ li {
 #toolbar {
```

Comment:
- [ ]  IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse. It's much preferred to use classes in selectors and then apply a class to an element in the page